index.js

```diff
@@ -31,26 +34,40 @@ const getWslDrivesMountPoint = (() => {
 
 		let isConfigFileExists = false;
 		try {
-			await pAccess(configFilePath, fs.constants.F_OK);
+			await fs.access(configFilePath, fs.constants.F_OK);
 			isConfigFileExists = true;
-		} catch (_) {}
+		} catch {}
 
 		if (!isConfigFileExists) {
 			// Default value for "root" param
 			// according to https://docs.microsoft.com/en-us/windows/wsl/wsl-config
 			return '/mnt/';
 		}
 
-		const configContent = await pReadFile(configFilePath, {encoding: 'utf8'});
+		const configContent = await fs.readFile(configFilePath, {encoding: 'utf8'});
 
-		mountPoint = (/root\s*=\s*(.*)/g.exec(configContent)[1] || '').trim();
+		mountPoint = (/root\s*=\s*(?<mountPoint>.*)/g.exec(configContent)?.groups?.mountPoint || '').trim();
 		mountPoint = mountPoint.endsWith('/') ? mountPoint : mountPoint + '/';
 
 		return mountPoint;
 	};
 })();
 
-module.exports = async (target, options) => {
+const pTryEach = async (array, mapper) => {
```

Comment:
Separate module?

Reply:
How about making it a part of AggregateError?

```js
await AggregateError.tryEach(array);
```